app/profile/[[...username]]/page.tsx

```diff
@@ -43,15 +51,22 @@ const Profile = async ({ params }: { params: { username?: string[] } }) => {
     ? params.username[0]
     : await getCurrentUsername();
 
-  const user = await getUser();
+  const profile = await getUserProfile(username);
+
+  const followers = profile.followers ? profile.followers.total : 0;
```

Comment:
```suggestion
  const followers = profile.followers?.total ?? 0;
```
This is more compact